include/nccl_ofi_rdma.h

```diff
@@ -403,8 +414,13 @@ typedef struct nccl_net_ofi_rdma_send_comm {
 	 * and `num_init_rails' is adjusted. */
 	int num_init_rails;
 
+#if HAVE_NVTX_TRACING
```

Comment:
shouldn't this be guarded by `HAVE_NVTX_TRACING && NCCL_OFI_NVTX_TRACE_PER_COMM`?

Reply:
Guarding the definitions is contingent on figuring out https://github.com/aws/aws-ofi-nccl/pull/377#discussion_r1556372811.